autowiring/CoreContext.h

```diff
@@ -1137,75 +1137,6 @@ class CoreContext:
   /// <param name="reference">A deferrable autowiring which may be used in type detection</param>
   MemoEntry& FindByDeferrableAutowiring(DeferrableAutowiring* deferred);
 
-  template<class T>
-  class AutoSearchLambdaAutowire:
-    public AutoSearchLambda
-  {
-  public:
-    AutoSearchLambdaAutowire(CoreContext& ctxt, AnySharedPointerT<T>& ref, DeferrableAutowiring& defer) :
-      ctxt(ctxt),
-      ref(ref),
-      defer(defer)
-    {}
-
-    CoreContext& ctxt;
-    AnySharedPointerT<T>& ref;
-    DeferrableAutowiring& defer;
-
-    void operator()() const override {
-      // If the slot wasn't autowired, complete the satisfaction here
-      if(!ref)
-        ctxt.AddDeferredAutowireUnsafe(&defer);
-    }
-  };
-
-  /// <summary>
-  /// Registers a slot to be autowired
-  /// </summary>
-  /// <param name="ref">The space where the resolved shared pointer will be stored</param>
-  /// <param name="defer">
-  /// In the event that resolution was not successful, the deferrable that will be used to perform delayed satisfaction
-  /// </param>
-  template<class T>
-  void Autowire(AnySharedPointerT<T>& ref, DeferrableAutowiring& defer) {
```

Comment:
This function is still called on line 105 of autowiring/Autowired.h, leading to a build error on all systems. Removing that clause reveals other compilation issues.